web/src/hooks/useTokens.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSprinterTokens } from '@chainsafe/sprinter-react'
+import { useMemo } from 'react'
+
+export const useTokens = () => {
+  const {
+    tokens: { data, error, loading }
+  } = useSprinterTokens()
+
+  const tokens = useMemo(() => {
+    return (data ?? []).concat({
+      name: 'ethereum',
```

Comment:
reason for hardcoding this?

Reply:
do you think it'd be nicer if you do something like

```typescript
// * Not provided by SDK
const MAINNET = {
      name: 'ethereum',
      decimals: 18,
      symbol: 'ETH',
      logoURI: 'https://scan.buildwithsygma.com/assets/icons/evm.svg',
      addresses: []
}
```

and then 
```typescript
(data ?? []).concat(MAINNET)
```

Reply:
I added it in [4c4bb2a](https://github.com/sprintertech/sprinter-sdk/pull/138/commits/4c4bb2acfb800d8a0a716aa8fe99ed3a8c8528da)
BTW it's not "mainnet", it's the native ETH token.